src/constants/links.ts

```diff
@@ -1 +1,7 @@
 export const SUGGEST_FEATURE_LINK = 'https://grill.hellonext.co'
+
+const SUGGEST_NEW_CHAT_ROOM_LINK =
```

Comment:
We had a misunderstanding. I proposed put hub name/id, but you did predefined chat name, so it is also cool! So lets do last update: predefine hub and chat :)

https://docs.google.com/forms/d/e/1FAIpQLSdhp3ZGAH3Gxbm6xtRcd8QgjI3M2RVdUY9UqepseXKr3DXirw/viewform?usp=pp_url&entry.96999585=HUB-NAME-OR-ID&entry.1674564644=SEARCH-RESULT